client/ui/profile.js

```diff
@@ -234,6 +228,56 @@ export default class UIProfile {
     this.backpackItems.appendChild(fragment);
   }
 
+  setItemActionMenu(isInEquipment) {
+    this.equipmentItemActions.textContent = "";
+
+    const fragment = new DocumentFragment();
+
+    const itemNameSpan = document.createElement("span");
+    itemNameSpan.classList.add("equipment__item-name");
+    itemNameSpan.innerText = this.selectedItemName;
+    fragment.appendChild(itemNameSpan);
+
+    const btnClose = createBtn({
+      classNames: actionClassNames,
+      datasets: [{ name: "actionName", value: ITEM_ACTIONS.CLOSE }],
+      text: "close",
```

Comment:
Maybe `cancel` instead?

Reply:
IMO both are ok. What is wrong with close?